docs/manifest.json

```diff
@@ -164,6 +164,15 @@
             ]
         },
         {
+            "id": "cloud-videointelligence",
+            "name": "google/cloud-videointelligence",
+            "defaultService": "videointelligence/videointelligenceserviceclient",
+            "versions": [
+                "v0.1.0",
```

Comment:
Yes, as Dave said, the tag will be added, so if it already exists there would be two entries for v0.1.0. The contents of VERSION are overwritten, so it's fine that that file is there.